### PR TITLE
Chat message - remove 'm_' prefix

### PR DESCRIPTION
**Pull request by DanVanAtta**
- move second class declaration up 'ChatMessage.java' to its own class file
- remove 'm_' prefix from both chat.java and newly created ChatMessage.java

---
This change is [Reviewable](https://reviewable.io/reviews/triplea-game/triplea/910)

### CODE REVIEW

File: src/games/strategy/engine/chat/ChatMessage.java [hunk 1]
```diff
@@ -0,0 +1,26 @@
+package games.strategy.engine.chat;
+
+
+public class ChatMessage {
+	private final String message;
+	private final String from;
+	private final boolean isMessage;
```
Comment (ron-murhammer) [inline src/games/strategy/engine/chat/ChatMessage.java]: Not sure if this is a good variable name change. I'm guessing that boolean is referring to whether the message came from the current client (not whether it is a message). Probably should be "isMyMessage".

**Comment by DanVanAtta** [comment]
updated

File: src/games/strategy/engine/chat/Chat.java [hunk 2]
```diff
@@ -269,14 +269,14 @@ public void chatOccured(final String message) {
...
-        for (final IChatListener listener : m_listeners) {
+      synchronized (mutexNodes) {
+        chatHistory.add(new ChatMessage(message, from.getName(), false));
+        for (final IChatListener listener : listeners) {
           listener.addMessage(message, from.getName(), false);
         }
         // limit the number of messages in our history.
-        while (m_chatHistory.size() > 1000) {
-          m_chatHistory.remove(0);
+        while (chatHistory.size() > 1000) {
```
Comment (RoiEXLab) [inline src/games/strategy/engine/chat/Chat.java]: Magic value! Since its commented I don't see a big need for this to be changed...

**Comment by RoiEXLab** [comment]
@DanVanAtta looks good now. I see a lot of Anonymous inner runnable classes. You either should replace those in this PR or mass replace all of them in another PR

**Pull request merged**